runelite-client/src/main/java/net/runelite/client/util/WinApi.java

```diff
@@ -0,0 +1,38 @@
+package net.runelite.client.util;
+
+import com.sun.jna.Native;
+import com.sun.jna.WString;
+import com.sun.jna.platform.win32.WinBase;
+import com.sun.jna.platform.win32.WinDef;
+import com.sun.jna.win32.StdCallLibrary;
+import java.time.LocalDateTime;
+
+interface Kernel32 extends StdCallLibrary
+{
+	int TIME_NOSECONDS = 0x00000002;
+
+	int GetTimeFormatEx(
+			WString lpLocaleName, WinDef.DWORD dwFlags, WinBase.SYSTEMTIME lpTime,
```

Comment:
you can use `int` instead of `WinDef.DWORD`, which should save some allocations and make the code slightly simpler.